lib/windows/app/components/SerialPortSelector.jsx

```diff
@@ -175,6 +185,7 @@ SerialPortSelector.propTypes = {
     dropdownCssClass: PropTypes.string,
     dropdownMenuCssClass: PropTypes.string,
     menuItemCssClass: PropTypes.string,
+    filterPort: PropTypes.func.isRequired,
```

Comment:
Would `filterPorts` be a more precise name here? The function receives an array of ports to be filtered.